test/resources/config/regexp.bal

```diff
@@ -0,0 +1,24 @@
+function name() {
+
+    string pattern = "ABC";
+    
+    string:RegExp _ = re `abc`;
+    string:RegExp _ = re `(abc)`;
+    string:RegExp _ = re `[abc]`;
+
+    string:RegExp _ = re `\p{gc=Ll}|\P{Lm}|\p{sc=Latin}`;
+    string:RegExp _ = re `\u{12FFFF}`;
+
+    string:RegExp _ = re `${pattern}`;
```

Comment:
Another possible test case is to use a function call within the interpolation. Can verify the function call expression's brackets - () are appropriately highlighted.

E.g.:
```
function myFn() returns string {
   return "abc";
}
```

```
string:RegExp _ = re `${myFn()}`;
```